api/rest/participants.py

```diff
@@ -0,0 +1,67 @@
+from datetime import date
+from rest_framework import serializers, status, validators, viewsets
+from rest_framework.response import Response
+
+from ..models import Participant, Team
+
+
+def is_true(value):
+    if not value:
+        raise serializers.ValidationError('Field is required')
```

Comment:
I find this error message a bit confusing. I would suggest something like "Must be checked" or "Must be true".